operations/iuf/workflows/management_rollout.md

```diff
@@ -47,6 +51,8 @@ procedures based on whether or not CSM is being upgraded:
 
 ### 3.1 `management-nodes-rollout` with CSM upgrade
 
+**`NOTE`** This subsection is optional and can be skipped if Install or upgrade additional products with IUF.
```

Comment:
This statement is not true.   If USS or SHS change you need to do `management-nodes-rollout`.

Reply:
Thanks for your comment . Note may not be added to 3.1 `management-nodes-rollout` with CSM upgrade
and 3.2 `management-nodes-rollout` without CSM upgrade